lib/process.js

```diff
@@ -68,10 +70,19 @@ function processTemplate(template, filename, config) {
 
   logResultErrors(result)
 
-  const babelify = transform(result.code, { filename: 'file.js' })
+  const tsconfig = getTsJestConfig(config)
 
-  return {
-    code: babelify.code
+  if (tsconfig) {
+    // they are using TypeScript.
+    const { outputText } = transpileModule(result.code, { tsconfig })
+    return { code: outputText }
```

Comment:
If it is a combination of ts and Babel, I think you should do transform twice, once with typescript and once with Babel. Should it be like that ?

Reply:
Hm, I am not sure in this particular case it makes any difference. Not we are transforming the code generated by the `@vue/sfc-compiler`, not an actual human, so I am not sure we need to care about polyfills. I could be wrong.